### PR TITLE
Park/unpark support (take 2)

### PR DESCRIPTION
**Pull request by dmlloyd**

### CODE REVIEW

File: java.base/src/main/java/java/lang/Thread$_patch.java [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    ThreadLocal.ThreadLocalMap inheritableThreadLocals;
+    long stackSize;
+    long tid;
+    int threadStatus;
+
+    // alias methods
+    native void setPriority(final int priority);
+    static native long nextThreadID();
+
+    // used only by Linux
```
Comment (dgrove-oss) [inline java.base/src/main/java/java/lang/Thread$_patch.java]: I think the comment is backwards?  Used only when not Linux?

File: java.base/src/main/resources/META-INF/qbicc/qbicc-patch-info [hunk 2]
```diff
@@ -2,4 +2,6 @@
 
 jdk.internal.loader.BootLoader$_patch
 java.lang.Runtime$_runtime
+java.lang.Thread$_patch
 jdk.internal.misc.Unsafe$_runtime
+jdk.internal.misc.UnsafeParkUnpark
```
Comment (dgrove-oss) [inline java.base/src/main/resources/META-INF/qbicc/qbicc-patch-info:8]: Also add `jdk.internal.misc.UnsafeThreadAccess` to patch-info?

**Review by dgrove-oss — APPROVED** [review]
Two small comments, but otherwise makes sense to me.

File: java.base/src/main/java/java/lang/Thread$_patch.java (hunk 1 above)
Comment (dmlloyd) [reply java.base/src/main/java/java/lang/Thread$_patch.java]: Fixed.

File: java.base/src/main/resources/META-INF/qbicc/qbicc-patch-info (hunk 2 above)
Comment (dmlloyd) [reply java.base/src/main/resources/META-INF/qbicc/qbicc-patch-info:8]: Fixed.

**Comment by dmlloyd** [comment]
Looks like I have a mix of an older implementation. Sorting it out now...

**Comment by dmlloyd** [comment]
Nope, it's OK.

**Comment by dmlloyd** [comment]
This is ready for review, though technically it won't actually work until #77 is merged.

**Pull request merged**